api/api.go

```diff
@@ -338,6 +338,7 @@ func (api *API) setupRoutes() {
 			network.POST("/new", api.createHostedIPFSNetworkEntryInDatabase)
 			network.POST("/stop", api.stopIPFSPrivateNetwork)
 			network.POST("/start", api.startIPFSPrivateNetwork)
+			network.DELETE("/remove", api.removeIPFSPrivateNetwork)
```

Comment:
ah oops did not notice this was missing at all 😅

Reply:
All good haha!